js/load-image.all.min.js

```diff
@@ -1,2 +1 @@
-//# sourceMappingURL=load-image.all.min.js.map
```

Comment:
No newline at end of file

Reply:
There is actually a newline, try editing the file and you'll see, I don't know why diff on github shows it like this